src/utils/index.ts

```diff
@@ -248,6 +264,150 @@ export interface ICurrentStats {
   };
 }
 
+export interface ICirculatingSupply {
+  heldByMultisig: BigNumber;
+  heldByUBQINK: BigNumber;
+  heldByGRANSUBQ: BigNumber;
+  heldByUBQESCH: BigNumber;
+  total: BigNumber;
+}
+
+export interface IDaoHoldings {
+  ubq: BigNumber;
+  ink: BigNumber;
+  esch: BigNumber;
+  lp: {
+    ubqInk: BigNumber;
+    gransInk: BigNumber;
+    inkEsch: BigNumber;
+  };
+}
+
+export const getDaoHoldings = async (provider: provider): Promise<IDaoHoldings> => {
+  // Get coin and token holdings from the DAO multisig address and farming address
+  const ubqHoldings = (await getCoinBalanceAsBigNum(provider, DAO_MULTISIG)).plus(await getCoinBalanceAsBigNum(provider, DAO_FARMING));
+  const inkHoldings = (await getBalanceAsBigNum(provider, INK, DAO_MULTISIG)).plus(await getBalanceAsBigNum(provider, INK, DAO_FARMING));
+  const eschHoldings = (await getBalanceAsBigNum(provider, ESCH, DAO_MULTISIG)).plus(await getBalanceAsBigNum(provider, ESCH, DAO_FARMING));
+
+  // Get lp holdings from the DAO multisig address and farming address
+  const ubqInkHoldings = (await getBalanceAsBigNum(provider, INK_UBQ_LPAddress, DAO_MULTISIG)).plus(
```

Comment:
ideally for the future we'd make this so it's an array of the values we want to acquire. and do a 'foreeach' of the elements in the array, so the 'get' code doesn't have to be modified but the array does.